SoftLayer/CLI/object_storage/list_accounts.py

```diff
@@ -9,12 +9,16 @@
 
 
 @click.command()
+@click.option('--limit',
+              type=int,
+              default=10,
```

Comment:
I think remove this, only should have a default parameter 
`def list_accounts(self, limit=10):`

Reply:
Having limit be a click option is what is needed. That allows users to change the limit when running the command.